bin/slirp

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+Dir.chdir(File.dirname(__dir__))
+
+require_relative '../lib/auto_hck'
+
+module AutoHCK
+  # rubocop:disable Metrics/BlockLength
+  run do
+    # rubocop:enable Metrics/BlockLength
+
+    require 'optparse'
+    require 'socket'
+
+    t = nil
```

Comment:
Can we name it better?

Reply:
I named it after the command line option. The meaning should be obvious if you know the option.

Reply:
Renamed the variable to `workspace_path` to match with the rest of the codebase. The command line option itself has also renamed to match with `bin/auto_hck`'s `w` option.